BrowserKit/Sources/Common/Theming/DarkTheme.swift

```diff
@@ -91,4 +91,20 @@ private struct DarkColourPalette: ThemeColourPalette {
 
     // MARK: - Shadow
     var shadowDefault: UIColor = FXColors.DarkGrey90.withAlphaComponent(0.16)
+    
+    // MARK: - Snackbar
```

Comment:
Ok so before moving further with the PR, we can make one adjustment already. We shouldn't add any new colors to the `ThemeColourPalette`. We have many tasks remaining before we can remove entirely the `LegacyTheme` protocol. 

For this task, we just want to focus on the snackbar classes, so the scope of the PR will be much smaller. Could you make those adjustments? If you're not sure where to start/stop let me know I can give further details. Also, for this sort of conversation it can be worthwhile to chat on [Element](https://chat.mozilla.org/#/room/#fx-ios:mozilla.org). You can write me a DM on there, I have the same username